docs/src/visualizations.md

```diff
@@ -65,12 +65,13 @@ fig
 
 And here with some extra tag metadata.
 
-```@example vis
-tag!(net[2,3], :specialplace, 1, 2)
-tag!(net[2,3], :otherdata, 3, 4)
-QuantumSavory.showmetadata(fig,ax,plt,2,3)
-fig
+<!-- ```@example vis -->
+# tag!(net[2,3], :specialplace, 1, 2)
+# tag!(net[2,3], :otherdata, 3, 4)
+# QuantumSavory.showmetadata(fig,ax,plt,2,3)
+# fig
 ```
+-->
```

Comment:
There are two closing tags here (`-->`) -- have you check how this renders in the docs -- I think it will actually wreck most of this page.

The `#` to comment out julia code are probably not necessary -- they will probably render as markdown-to-html h1 tags